docker_test.go

```diff
@@ -136,6 +136,97 @@ func TestContainerWithNetworkModeAndNetworkTogether(t *testing.T) {
 	}
 }
 
+func TestContainerWithHostNetworkOptionsAndWaitStrategy(t *testing.T) {
+	ctx := context.Background()
+	gcr := GenericContainerRequest{ContainerRequest: ContainerRequest{
+		Image:       "nginx",
+		SkipReaper:  true,
+		NetworkMode: "host",
+		WaitingFor:  wait.ForListeningPort("80/tcp"),
+	},
+		Started: true,
+	}
+
+	nginxC, err := GenericContainer(ctx, gcr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer nginxC.Terminate(ctx)
+
+	host, err := nginxC.Host(ctx)
+	if err != nil {
+		t.Errorf("Expected host %s. Got '%d'.", host, err)
+	}
+
+	_, err = http.Get("http://" + host + ":80")
+	if err != nil {
+		t.Errorf("Expected OK response. Got '%d'.", err)
+	}
+}
+
+func TestContainerWithHostNetworkAndEndpoint(t *testing.T) {
+	nginxPort := "80/tcp"
+	ctx := context.Background()
+	gcr := GenericContainerRequest{ContainerRequest: ContainerRequest{
+		Image:       "nginx",
+		SkipReaper:  true,
+		NetworkMode: "host",
+		WaitingFor:  wait.ForListeningPort(nat.Port(nginxPort)),
+	},
+		Started: true,
+	}
+
+	nginxC, err := GenericContainer(ctx, gcr)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer nginxC.Terminate(ctx)
+
+	hostN, err := nginxC.Endpoint(ctx, "")
+	if err != nil {
+		t.Errorf("Expected host %s. Got '%d'.", hostN, err)
+	}
+	t.Log(hostN)
+
+	_, err = http.Get("http://" + hostN)
+	if err != nil {
+		t.Errorf("Expected OK response. Got '%d'.", err)
+	}
+}
+
+func TestContainerWithHostNetworkAndPortEndpoint(t *testing.T) {
+	nginxPort := "80/tcp"
+	ctx := context.Background()
+	gcr := GenericContainerRequest{ContainerRequest: ContainerRequest{
+		Image:       "nginx",
+		SkipReaper:  true,
+		NetworkMode: "host",
+		WaitingFor:  wait.ForListeningPort(nat.Port(nginxPort)),
+	},
```

Comment:
I guess this format is not correct, could you fix that? 🙏

Reply:
Only format?! It's not a problem to fix it. :ok_hand: 
This a strange that **go fmt** does not reformat a code.